client/my-sites/post-type-list/post-actions-ellipsis-menu/qrcode.jsx

```diff
@@ -0,0 +1,73 @@
+import { Icon } from '@wordpress/components';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import { Component } from 'react';
```

Comment:
I _think_ that we can import this from the `@wordpress` package thing.

Reply:
Done!